src/test/groovy/hudson/plugins/gradle/GradlePluginIntegrationTest.groovy

```diff
@@ -271,6 +272,11 @@ task hello << { println 'Hello' }"""))
         installation.name == 'myGradle'
         installation.home == '/tmp/foo'
 
+        def envVars = new EnvVars()
+        installation.buildEnvVars(envVars)
+        assert envVars.containsKey("PATH+GRADLE")
+        assert envVars.get("PATH+GRADLE") == installation.home + "/bin"
```

Comment:
By an integration test I rather meant that the path to `GRADLE` is really added to the path and then we can execute gradle. Preferable in a pipeline job. Is that possible?